pkg/provider/acceptance_source_kafka_migration_test.go

```diff
@@ -0,0 +1,111 @@
+package provider
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccSourceKafkaMigration_basic(t *testing.T) {
```

Comment:
Still working on this. Still working on finding better examples of doing the migration tests.